scripts/regclient/deps.lua

```diff
@@ -44,6 +45,9 @@ deps.depImages = {
   "ghcr.io/pixie-io/nats@sha256:55521ffe36911fb4edeaeecb7f9219f9d2a09bc275530212b89e41ab78a7f16d",
   "ghcr.io/pixie-io/python_grpc_1_19_0_helloworld@sha256:e04fc4e9b10508eed74c4154cb1f96d047dc0195b6ef0c9d4a38d6e24238778e",
   "ghcr.io/pixie-io/python_mysql_connector@sha256:ae7fb76afe1ab7c34e2d31c351579ee340c019670559716fd671126e85894452",
+
+  -- non build dependency list
+  "gcr.io/pixie-oss/pixie-dev-public/curl:multiarch-7.87.0@sha256:f7f265d5c64eb4463a43a99b6bf773f9e61a50aaa7cefaf564f43e42549a01dd",
```

Comment:
hmm, we also have a curl with sha `5594e102d5da87f8a3a6b16e5e9b0e40292b5404c12f9b6962fd6b056d2a4f82` (see above).

should we figure out which one is newer and only keep one of them?

Reply:
The pixie multiarch one is newer, so let's migrate to that one.
```
$ docker run docker.io/curlimages/curl@sha256:5594e102d5da87f8a3a6b16e5e9b0e40292b5404c12f9b6962fd6b056d2a4f82 --version
curl 7.74.0-DEV (x86_64-pc-linux-musl) libcurl/7.74.0-DEV OpenSSL/1.1.1g zlib/1.2.11 libssh2/1.9.0 nghttp2/1.40.0
Release-Date: [unreleased]
Protocols: dict file ftp ftps gopher http https imap imaps mqtt pop3 pop3s rtsp scp sftp smb smbs smtp smtps telnet tftp
Features: alt-svc AsynchDNS HTTP2 HTTPS-proxy IPv6 Largefile libz NTLM NTLM_WB SSL TLS-SRP UnixSockets
```

I've added a TODO to remove the older one once the multiarch one is in place and validated. I don't see any reason why it would cause issues with the bpf trace tests its used in, but I'd like to see it working before removing the older one.